VCFv4.3.tex

```diff
@@ -1423,7 +1434,7 @@ \subsubsection{Genotype encoding}
 
 The value is always implicitly a vector of N values, where N is the number of samples.  The type byte of the value field indicates the type of each value of the N length vector.  For atomic values this is straightforward (size = 1).  But if the type field indicates that the values are themselves vectors (as often occurs, such as with the PL field) then each of the N values in the outer vector is itself a vector of values.  This encoding is efficient when every value in the genotype field vector has the same length and type.
 
-Note that the specific order of fields isn't defined, but it's probably a good idea to respect the ordering as specified in the input VCF/BCF2 file.
+It is recommended to respect the ordering as specified in the input VCF/BCF2 file, but parsers should not rely on a specific ordering.
```

Comment:
It refers to the FORMAT field ordering. So, for example, in the example just above, BCF could in principle encode the GT:GQ:DP fields as DP:GQ:GT.